test/sql/test_selectable.py

```diff
@@ -1137,6 +1137,26 @@ def test_whereclause_adapted(self):
         assert s3._whereclause.left.table is not s1
         assert s3._whereclause.left.table in s3._froms
 
+    def test_values(self):
+        v = Values(
+            [
+                column("mykey", Integer),
+                column("mytext", String),
+                column("myint", Integer),
+            ],
+            (1, "textA", 99),
+            (2, "textB", 88),
+            alias_name="myvalues",
+        )
+        assert str(v) == "VALUES (1, 'textA', 99), (2, 'textB', 88)"
+        s = select([v])
+        expected = (
+            "SELECT mykey, mytext, myint \n"
+            "FROM (VALUES (1, 'textA', 99), (2, 'textB', 88)) "
+            "AS myvalues (mykey, mytext, myint)"
+        )
+        assert str(s) == expected
+
```

Comment:
**mike bayer** (zzzeek) wrote:

same

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1608

Reply:
**Gord Thompson** (gordthompson) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1608